apps/studio/src/components/CoreTabs.vue

```diff
@@ -593,6 +599,17 @@ export default Vue.extend({
       this.tableExportOptions = options
       this.showExportModal = true
     },
+    async alterTable(options: { changes: AlterTableSpec, table: TableOrView, onError?: (e: any) => void, onSuccess?: () => void }) {
+      try {
+        await this.connection.alterTable(options.changes)
+        await this.$store.dispatch('updateTableColumns', options.table)
```

Comment:
won't updating the table columns just force an update to the TableTable without an extra event? I think there's a watcher on the columns already no?

Reply:
Yeah but for some reason it didn't detect the changes automatically, even after calling the updateColumnsTable

Reply:
Wait, let me find why the watcher didn't work.